frontend/src/fetchers/recipientSpotlight.js

```diff
@@ -5,7 +5,7 @@ import { get } from './index';
 const recipientUrl = join('/', 'api', 'recipient-spotlight');
 
 export const getRecipientSpotlight = async (sortBy = 'recipientName', sortDir = 'desc', offset = 0, filters, limit = 10) => {
-  const limitParam = limit !== undefined ? `&limit=${limit}` : '';
+  const limitParam = limit !== undefined && limit !== null ? `&limit=${limit}` : '';
   const queryString = `?sortBy=${sortBy}&direction=${sortDir}&offset=${offset}${limitParam}${filters ? `&${filters}` : ''}`;
   const spotlight = await get(`${recipientUrl}${queryString}`);
```

Comment:
We could probably refactor this to use the URLParams constructor, it might be easier to read.
Also, can limit be undefined if we pass in a default parameter?

```js
const params = new URLParams();

if(limit) {
  params.set('limit', limit);  
}

params.set('sortBy', sortBy);

// etc with offset & filters

const queryString = params.toString();
const spotlight = await get(`${recipientUrl}${queryString}`);

Reply:
Im not seeing a simple ternary as being hard to read.